src/test/java/org/salesforce/steps/CreateCampaignSteps.java

```diff
@@ -0,0 +1,54 @@
+package org.salesforce.steps;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+import salesforce.ui.PageTransporter;
+import salesforce.ui.pages.LoginPage;
+import salesforce.ui.pages.lightning.campaign.CampaignCreatedPage;
+import salesforce.ui.pages.lightning.campaign.CampaignPage;
+import salesforce.ui.pages.lightning.campaign.CreateCampaignPage;
+
+public class CreateCampaignSteps {
+
+    private CampaignPage campaignPage;
+    private CampaignCreatedPage campaignCreatedPage;
+    protected LoginPage loginPage;
+    private PageTransporter pageTransporter;
+    String campaignName = "New Campaign Created";
+
+    public CreateCampaignSteps(PageTransporter pageTransporter) {
+        this.pageTransporter = pageTransporter;
+    }
+
+    @Given("I navigate to {string} page")
+    public void iNavigateToPage(String arg0) {
+        pageTransporter.navigateToFeaturePage(arg0);
+    }
+
+    @When("I create a dateof Campaign with fields")
+    public void iCreateADateofCampaignWithFields() {CreateCampaignPage createCampaignPage = campaignPage.clickCreateCampaignBtn();
+        createCampaignPage.setInputField("campaign name", campaignName);
+        createCampaignPage.clickActive();
+        campaignCreatedPage = createCampaignPage.clickSaveBtn();
+        campaignCreatedPage.clickDetailTab();
+        campaignCreatedPage.waitElementCampaignNameCreated();
+    }
+
+    @Then("A successful message of creating is display")
+    public void aSuccessfulMessageOfCreatingIsDisplay() {
+        Assert.assertEquals(campaignCreatedPage.getTextAlertSuccess(), "success\nCampaign " + "\"" + campaignName + "\"" + " was created.\nClose", "Campaign was not created");
```

Comment:
It could be better to use soft asserts instead of hard asserts, and AssertAll on Test's post conditions

Reply:
We will do it later.